index.html

```diff
@@ -70,5 +70,218 @@ <h2>LET'S CONNECT</h2>
+        <h1>About Myself</h1>
+    </section>
+    <section class="about-headline-description">
+        <p>
+                I'm a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience!
+            If you like what you see and have a project you need coded,
+            don't hesitate to contact me.
+        </p>
+    </section>
+    <section class="about-social-media">
+        <h2>LET'S CONNECT</h2>
+    </section>
+      <ul class="about-social-media-icons">
+        <li>
+          <a href="#">
+            <img src="./Properties/Twitter.png" alt="twitter">
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="./Properties/Linkedin icon.png" alt="linkedin">
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="./Properties/Mail.png" alt="mail">
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="./Properties/Github.png" alt="github">
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="./Properties/Vector.png" alt="vector">
+          </a>
+        </li>
+      </ul>
+      <button class="user-buttons">
+        Get my resume
+      </button>
+
+      <section class="languages-frameworks">
+        <ul class="languages-frameworks-list">
+          <li>
+            <article class="language-title">
+              <h2>Languages</h2>
+              <img src="./Properties/down-arrow.svg" alt="">
+            </article>
+            <article class="language-name">
+              <section class="language-logo">
+                <img src="./Properties/js.png" alt="">
+              </section>
+              <section class="certain-language-name">
+                <p>JavaScript</p>
+              </section>
+            </article>
+            
+            <article class="language-name">
+              <section class="language-logo">
+                <img src="./Properties/html.png" alt="">
+              </section>
+              <section class="certain-language-name">
+                <p>JavaScript</p>
+              </section>
+            </article>
+
+            <article class="language-name">
+              <section class="language-logo">
+                <img src="./Properties/css.png" alt="">
+              </section>
+              <section class="certain-language-name">
+                <p>JavaScript</p>
+              </section>
+            </article>
+          </li>
+          <li>
+            <article class="framework-title">
+              <h2>Frameworks</h2>
+              <img src="./Properties/right-arrow.svg" alt="">
+            </article>
+          </li>
+          <li>
+            <article class="skill-title">
+              <h2>Skills</h2>
+              <img src="./Properties/right-arrow.svg" alt="">
+            </article>
+          </li>
+        </ul>
+      </section>
```

Comment:
- [ ] You should stick to the design as much as possible like color, font, spacing etc. Please use same color for `javascript, javascript, javascript,`.
- [ ] [Optional] Also make `javascript, javascript, javascript, and change these 2 into HTML and CSS.
- [ ] I would suggest you please make the list language, framework and skills that looks like figma deisgn. Please decrease the size.
- [ ] Down arrow also looks different please make similar to the design.
### Your design
![his](https://user-images.githubusercontent.com/81727988/235164287-c4c43c96-b72a-4dab-8a06-92a367389dad.png)

### Figma design
![fimga](https://user-images.githubusercontent.com/81727988/235164349-d53674fe-c4f4-4dce-9249-e15f37599833.png)